harmonyfrontend/src/components/library/series-list.jsx

```diff
@@ -49,8 +50,12 @@ export default observer(
         return <Loading />;
       }
 
+      let seriesTitles = Object.keys(series).sort((a, b) => {
+        return sortableTitle(a) > sortableTitle(b) ? 1 : -1;
```

Comment:
I like the idea of using an intelligent title sort, this is great but @dylanpizzo the code should be moved src/model/library.js wth a new function called 
```
sortTitles(titles)
```
or something of that sort which takes an array of strings and returns them sorted using the title sort rules you define.